tox.ini

```diff
@@ -247,6 +247,17 @@ passenv =
     SSH_AUTH_SOCK
     OS_*
 
+[testenv:integration-tests-fast]
+deps = 
+    -r{toxinidir}/integration-requirements.txt
+    -r{toxinidir}/test-requirements.txt
+commands = {envpython} -m pytest --log-cli-level=INFO -m "not hypothesis_slow" -m "not serial" --cov=cloudinit {posargs:tests/integration_tests}
```

Comment:
How are multiple workers being used here?

Also, this isn't the right way to include coverage as the code is being executed on remote systems. See https://github.com/canonical/cloud-init/blob/3f821536ec2344009871091b6c572b8f070a16ca/tests/integration_tests/integration_settings.py#L100-L105 . Coverage also shouldn't be enabled by default as its too invasive in integration tests.

Reply:
Alright I'm down to not include coverage here. This was more of a lazy copy pasta than anything. Thanks.